tests/lib/rules/use-waiters-in-module-scope.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const rule = require('../../../lib/rules/use-waiters-in-module-scope');
+
+const { ERROR_MESSAGE } = rule;
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('use-waiters-in-module-scope', rule, {
+  valid: [
+    `
+    import { buildWaiter } from 'ember-test-waiters';
+
+    let myWaiter = buildWaiter('waiterName');
+  `,
+    `
+    import { buildWaiter } from 'table-waiters';
+
+    function useWaiter() {
+      let myOtherWaiter = buildWaiter('the second');
+    }
+  `,
```

Comment:
There should be valid test cases for:

```
import { buildWaiter } from 'ember-test-waiters';

function useWaiter() {
  let myWaiter = somethingElse.buildWaiter('waiterName');
}
```

```
import { buildWaiter } from 'ember-test-waiters';

function useWaiter() {
  let myWaiter = buildWaiter.somethingElse('waiterName');
}
```

Reply:
Makes sense, for completeness. I seem to always be able to count on your for permutations :D

Reply:
I've added these.